en/getting-started/autodisco-aws.md

```diff
@@ -0,0 +1,83 @@
+---
+id: autodisco-aws
+title: Discover resources automatically
+---
+
+In this tutorial, we will use the [Auto Discovery module](../monitoring/discovery/hosts-discovery.html) to detect AWS EC2 instances and monitor them.
+
+## Prerequisites
+
+You must have the credentials to the AWS group of instances you want to monitor (Name, AWS Acces Key and AWS Secret Key), and the region your AWS instances are in.
+
+## Step 1: Installing the Amazon EC2 plugin
+
+1. Make sure your [license](../administration/licenses.html) is installed. (The Auto Discovery module requires a valid license. It is included in the free [IT-100](IT100.html) license.)
+
+2. [Install](../monitoring/discovery/installation.html) the autodiscovery module.
+
+3. Make the Plugin Packs available:
+
+    - If you have an offline licence, in your terminal, enter:
+     
+        ```shell
+        yum install centreon-pack-*
+        ```
+
+    - If you have an online license, the Plugin Packs are already available in the interface.
+
+4. To install the Amazon EC2 Plugin Pack, go to **Configuration > Plugin Packs**.
+
+5. In the **Keyword** field, type **Amazon EC2** and then click **Search**.
+
+6. Click on the `+` sign in the top right corner of the Plugin Pack. The Plugin Pack now has a green border and a tick mark in the top right corner. 
+
+7. Follow [this procedure](../integrations/plugin-packs/procedures/cloud-aws-ec2.html) (sections [Plugin dependencies](../integrations/plugin-packs/procedures/cloud-aws-ec2.html#plugin-dependencies) and [Setup](../integrations/plugin-packs/procedures/cloud-aws-ec2.html#setup)) to finish installing the Plugin Pack. (You can also access the documentation for a Plugin Pack by clicking on the `i`.)
+
+    ![image](../assets/getting-started/aws-doc.png)
+
+## Step 2: Configure the discovery job
+
+3. Go to **Configuration > Hosts > Discovery**, and then click **Add**.
+
+4. Enter a name for your discovery job, and then select the **Amazon AWS EC2** provider. Click **Next**.
+
```

Comment:
The image below is not displayed in the jenkins preview because of the CAPS "PNG" in the filename